src/pdfpc.vala

```diff
@@ -44,7 +44,10 @@ namespace pdfpc {
         /**
          * Commandline option parser entry definitions
          */
-        const OptionEntry[] options = {
+        const OptionEntry[] options = {		 
+            {"external-script", 'X', 0, OptionArg.STRING,
```

Comment:
Please keep the ABC order of options (short variant)

Reply:
I fixed it. There was also another wrongly ordered item.